.github/workflows/ci.yml

```diff
@@ -40,22 +46,8 @@ jobs:
         run: make setup
 
       # Build artifacts
-      - name: Event name
```

Comment:
Why do we delete this job?

Reply:
I could put it in the `trunk` workflow actually, but I wanted to remove it from the build/tests CI. That being said, I'm not sure it's a good idea to have this since it doesn't enforce dev to have a proper local setup. With the trunk setup being one app/one command for all, I've guesses that it was better to enforce this than to patch at the CI level

Reply:
okay makes sense